source/data_analysis_spring/data_analysis_spring_2023.Rmd

```diff
@@ -1083,6 +941,275 @@ new_data |>
 
 
 
+```
+
+# Analyse tijdsreeksen
+
+## Wilde bijen
+
+```{r}
+# Add unique identifier for each sampling site and method combination
+# and find the number of species for the highest number of days
+apoidea_cumulative <- apoidea_cumulative |>
+  mutate(
+    sampling_site_method = paste0(
+      sampling_site_cd, "_", spring_code)) |>
+  group_by(sampling_site_method) |>
+  mutate(n_species_final = max(cumulative_n_species)) |>
+  ungroup() |>
+  mutate(prop = cumulative_n_species / n_species_final)
+```
+
+```{r}
+model5 <- glmmTMB(prop ~ cumulative_days * method_combi +
+                    (1 | sampling_site_method),
+                  family = binomial,
+                  weights = n_species_final,
+                  data = apoidea_cumulative)
+```
+
+```{r}
+# check model assumptions
+performance::check_model(model5)
+```
+
+```{r}
+car::Anova(model5)
+```
+
+Verwacht percentage van de "totale" gemeenschap dat gevonden wordt voor het aantal dagen dat pan traps geplaatst worden:
+```{r}
+emmeans::emmeans(model5, pairwise ~ cumulative_days,
```

Comment:
Ik denk dat je hier beter "pairwise" vervangt door "trt.vs.ctrl1" (zie https://rvlenth.github.io/emmeans/reference/emmc-functions.html) omdat we enkel geïnteresseerd zijn in verschillen t.o.v. de eerste waarde in de `at = list()`.